docs/MSPX.md

```diff
@@ -19,33 +22,92 @@ mLRS provides the MspX technology, which is designed to improve the over-the-air
 MspX is enabled by this setting in the receiver:
 
 - "Rx Ser Link Mode" = "mspX"
+- "Ser Baudrate" = "115200" or "230400" (for INAV 8 and later)
 
 #### MSP-RC
 
 For enabling MSP-RC set:
 
-- "Rx Snd RcChannel" = "rc override" or "rc channels"
+- "Rx Snd RcChannel" = "rc override" or "rc channels" (both have the same functionality in MspX mode)
 
-With MSP-RC enabled, SET_RAW_RC messages are sent to the flight controller. In the flight controller configure the receiver to the MSP protocol. This allows one to avoid the extra wire for CRSF or SBus. MspX needs to be enabled.
+***Notes***:
+- RC link statistics are only sent to the flight controller via MSP if "rc override" or "rc channels" is selected. 
+- MSP-RC messages will always override other RC Link inputs. If CRSF is connected and CRSF selected as the RC Protocol in INAV, MSP-RC still has priority if enabled in mLRS and connected to a MSP Enabled UART.
 
 ## INAV Configuration
 
-The baudrate should be set to 115200 as lower baudrates have shown irregular telemetry update rates on the radio due to the lack of flow control on the INAV side. 
-
-INAV 8.0 and later will also support a baudrate of 230400 but no change in performance or stability was noticed. Additionally with INAV 8.0 the performance of the serial link to a ground control station will be increased, due to some optimizations in INAV. 
+The baudrate should be set to 115200 or higher for consistent dataflow and RC control. Baudrates of less than 57600 can cause message loss on the INAV side and result in inconsistent RC control at 50Hz RC Rate, if MSP-RC is used (INAV Limitation).  
```

Comment:
Could this be a subbullet under the 'Set the baudrate to 115200 or higher' bullet?  Seems a bit odd to start with this line IMO.

Reply:
Yeah I must have missed that when I moved stuff around. I merged that with the configuration steps.